### ISSUE TITLE
[Question] PermissionError when using Docker

### ISSUE THREAD

**Issue by edugmes** [kolofordjango/kolo#12]
I installed kolo in my Django pip requirements and added Kolo to the start of middleware in settings, but when I access a page I get the following error:

```
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/kolo/profiler.py", line 225, in save_request_in_db
    subprocess.run(
  File "/usr/local/lib/python3.8/subprocess.py", line 493, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/local/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: 'git'
```

The page loads, but the error above appears. What am I supposed to do? Thanks in advance!

**Comment by wilhelmklopp** [comment]
@edugmes 👋 Thanks for reporting this!

Can I ask what your docker base image is? And is `git` installed in the container?

**Comment by Ian-Foote** [comment]
A fix for this will be available in the next version of Kolo. Instead of crashing, Kolo will instead record the git commit sha as `None`.

**Comment by Ian-Foote** [comment]
Making `git` available with the right permissions in docker would also resolve this, but that's not something we can do from Kolo's side.

**Comment by wilhelmklopp** [comment]
👋 @edugmes I just released 1.3.8 of the kolo python package which includes @Ian-Foote's fix 🎉 

Thanks for jumping on this right away @Ian-Foote 🙇‍♂️

**Comment by edugmes** [comment]
Thanks!  My docker image is python:3.8 (debian) and python:3.8-slim (also debian). Would you guys mind.
It seems that issue is that the docs weren't very clear about the requirements concerning docker. I'll install everything again and test it. Thanks for the replies.
